### PR TITLE
Pathlib support

### PR DESCRIPTION
**Pull request by fsoubelet**
Implementation for #26 and #27 (somehow the issue was duplicated at creation?)

### CODE REVIEW

**Comment by fsoubelet** [comment]
I believe this should make version `0.2` since we're adding functionality.

Do you agree @lmalina? This would require an additional commit to change `__version__`

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # SDDS-Files Changelog
 
+## Version 0.2
+- Added:
+    - Support for ascii files.
+    - Support for PosixPath objects from `pathlib` in read and write operations.
```
Comment (lmalina) [inline CHANGELOG.md:6]: just Path...

**Review by lmalina — CHANGES_REQUESTED** [review]
just details

**Review by lmalina — APPROVED** [review]
👍

**Pull request merged**